velox/exec/OutputBuffer.h

```diff
@@ -124,10 +161,36 @@ class DestinationBuffer {
   // The sequence number of the first item to pass to 'notify'.
   int64_t notifySequence_{0};
   uint64_t notifyMaxBytes_{0};
+  BufferInfo info_;
 };
 
 class Task;
 
+struct OutputBufferInfo {
+  enum class BufferState {
+    OPEN,
```

Comment:
naming: kOpen

please, check coding guidelines and update

Reply:
done.